### PR TITLE
sql: update tidb specific

### PR DESCRIPTION
**Pull request by CaitinChen**
via
https://github.com/pingcap/docs-cn/pull/654
https://github.com/pingcap/docs-cn/pull/658
@lilin90 @XuHuaiyu PTAL

### CODE REVIEW

File: sql/tidb-specific.md [hunk 1]
```diff
@@ -16,25 +16,197 @@ If you need to set the global variable, run:
...
+- Scope: SESSION
+- Default value: ""
+- This variable is used to set the time point at which the data is read by the session. For example, when you set the variable to "2017-11-11 20:20:20", the current session reads the data of this moment.
+
+### tidb_import_data
+
+- Scope: SESSION | GLOBAl
+- Default value: 0
+- This variable indicates whether to import data from the dump file currently.
+- When the variable is set to 1, the unique index constraint is not checked to speed up importing.
```
Comment (lilin90) [inline sql/tidb-specific.md]: Is it clearer to change this sentence to the following version?

"To speed up importing, the unique index constraint is not checked when the variable is set to 1."

File: sql/tidb-specific.md [hunk 2]
```diff
@@ -16,25 +16,197 @@ If you need to set the global variable, run:
...
+- Default value: 0
+- This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function to the position before Join.
+- When the aggregate operation is slow in query, you can set the variable value to 1.
+
+### tidb_opt_insubquery_unfold
+
+- Scope: SESSION
+- Default value: 0
+- This variable is used to set whether the optimizer executes the optimization operation of 
+unfolding the "in-" subquery.
```
Comment (lilin90) [inline sql/tidb-specific.md]: Please combine this line with the line above.

File: sql/tidb-specific.md [hunk 3]
```diff
@@ -16,25 +16,197 @@ If you need to set the global variable, run:
...
+
+### tidb_build_stats_concurrency
+
+- Scope: SESSION
+- Default value: 4
+- This variable is used to set the concurrency of executing the ANALYZE statement.
```
Comment (lilin90) [inline sql/tidb-specific.md]: Add inline code marks for ANALYZE.

File: sql/tidb-specific.md [hunk 4]
```diff
@@ -16,25 +16,197 @@ If you need to set the global variable, run:
...
+
+- Scope: SERVER
+- Default value: 0
+- This variable is used to set whether to record all the SQL statements in the log.
+
+### tidb_enable_streaming
+
+- Scope: SERVER
+- Default value: 0
+- This variable is used set whether to enable Streaming.
```
Comment (lilin90) [inline sql/tidb-specific.md]: used -> used to

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

**Comment by CaitinChen** [comment]
@XuHuaiyu  PTAL

**Pull request merged**